flow/table.go

```diff
@@ -120,11 +122,12 @@ func NewTable(updateHandler *Handler, expireHandler *Handler, nodeTID string, op
 		// convert seconds to milleseconds
 		appTimeout[strings.ToUpper(key)] = int64(1000 * config.GetConfig().GetInt("flow.application_timeout."+key))
 	}
-
+	LRU, _ := simplelru.NewLRU(500000, nil)
```

Comment:
This is now a strong limitation, while I'm fine with it I would add something in the config file to set it, default of 500k seems ok:

https://github.com/skydive-project/skydive/blob/master/etc/skydive.yml.default#L455

Reply:
I'm not sure about what advantage this implementation about LRU brings as internally it is a map, there is just a check that limit the amount of data and as there is a evictList it should take even more memory space. Probably limiting the amount of flows is better. Moreover there is no event when a flow is evicted so we can't mark it as timeout.

Reply:
Well, if you remember the previous implementation had no limit on the table's size which was causing it to inflate in size and kill the system. LRU implementation as we discussed enables you to ensure this doesn't happen and alliws you to keep the most recent flows as well. Also we can add a functionality to handle evictions if you'd like...

Reply:
Well, if you remember the previous implementation had no limit on the table's size which was causing it to inflate in size and kill the system. LRU implementation as we discussed enables you to ensure this doesn't happen and alliws you to keep the most recent flows as well. Also we can add a functionality to handle evictions if you'd like...